packages/terra-card/src/_mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin terra-consumer-theme-card() {
+  --terra-card-box-shadow: 0 1px 1px 0 #1c1f21;
```

Comment:
In our UI uplift the shadow is set to `0 1px 3px rgba(0, 0, 0, 0.3)`. I think I'd ask UX which is correct. Personally, I think a semi transparent shadow always looks better.